### ISSUE TITLE
64 eats cake

### ISSUE THREAD

**Issue by Tribblicious** [Tribblers/TestRepo01#1]
64 there needs to be cake

**Comment by 64FF00** [comment]
```PHP
This is the code
```

**Comment by Tribblicious** [comment]
Can you check sample

**Comment by HyGlobalHD** [comment]
do u guys want to help me? I'm currently developing a plugin for pocketmine if u want to see it
[click here](https://github.com/ArchRPG/ARCore)

**Comment by Tribblicious** [comment]
Yeah ok although I'm the noob who just started with developing but 64 is the tutor so sure XD

**Comment by HyGlobalHD** [comment]
If u want to see the basic there a simple template
at [this](https://github.com/rdcmcpe/rdc_plugins/blob/master/MyTemplates-master.zip)

**Comment by 64FF00** [comment]
@Tribblicious See https://github.com/PurePlugins/TapToDo-LEET.CC

**Comment by Tribblicious** [comment]
ok im trying to make an antikick plugin that checks a list, however server im helpling with is having issues with their core not being able to work properly
